docs/api.md

```diff
@@ -145,18 +145,19 @@ TestScenarioSpec defines the load testing scenario
 _Appears in:_
 - [GatlingSpec](#gatlingspec)
 
-| Field | Description |
```

Comment:
Did you update this file with the `make docs` command?
If not, please update it using `make docs`.

Reply:
this one was modified manually, thanks for spotting that, I did run `make docs` this time :)